client/src/pages/admin/CreateProduct.test.js

```diff
@@ -0,0 +1,32 @@
+import '@testing-library/jest-dom';
```

Comment:
For `CreateProduct.test.js`, is there a reason why you only checked for rendering and not the `handleCreate` and `getAllCategory` functions? 

Also, note that the `CreateProduct` component uses child components like `Layout`, `AdminMenu` and so on. You might need to mock these components, as we are doing unit testing and need to isolate the component under test.

Reply:
Thanks for the suggestion! I plan to test functional/method units as part of expanding these basic unit tests in the rest of the sprint.